src/test/integration/ProjectsClient.test.ts

```diff
@@ -26,6 +26,28 @@ describe("ProjectsClient", () => {
     chai.expect(projectList).to.not.be.empty;
   });
 
+  it("should get project details from projectId", async () => {
+    const projectDetails: Project[] = await projectsAccessClient.getAll(accessToken,
+      {
+        pagination: {
+          top: 1,
+        },
+      });
+
+    const mappedFullProjectList: Project[] = projectDetails.map((project) => {
```

Comment:
This step would only be necessary if you are using chai's .to.deep.include.members comparison. You can manually compare the id, name, and code properties instead since you only have one project.